intel-retail/automated-self-checkout
# refactor: ovms model download scripts refactor and cleanup

clean up the old model downloader and refactor the ovms model downloader to a better function structure in script

CLOSES: #409

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**409**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- for unit-test: 
   ```console
        cd ./download_models; ./testDownloadOVMSModels.sh
  ```
should see ALL PASSED
- for integration tests: run smoke tests `make run-smoke-tests`  and should see ALL PASSED

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
